modules/administration/pages/ptfs.adoc

```diff
@@ -0,0 +1,115 @@
+
+. In the {productname} {webui}, navigate to menu:Systems[Systems List] and select the client where you want to install a PTF.
+. Navigate to the menu:Systems[Software > Packages > Software Channels] and select the [systemitem]``PTF channel``.
+. Click btn:[Next], and [guimenu]``Confirm Software Channel Change`` with btn:[Confirm].
+. To check if the channel assignment is finished, navigate to menu:Systems[Events > History] to see the results.
+. Navigate to the menu:Systems[Software > PTFs > Install] sub tab.
+. Select the ptf package you want to install.
+. Click btn:[Install PTF], and [guimenu]``Confirm Program Temporary Fixes (PTFs) Installation`` with btn:[Confirm].
+. To see PTF installation results, navigate to menu:Systems[Events > History].
+
+
+In case a PTF should be installed using the API, the normal [systemitem]``system.schedulePackageInstall`` API can be used with the proxy package name.
+
+
+== After PTF installation
+
+Once a PTF is confirmed to address the reported issue, the updated package will be tracked for inclusion
+into a future maintenance update before being widely distributed as an regular maintenance update in the update repositories.
+
+When this regular update with the fix is released, an updated version of the PTF will also be released into
+the account-specific PTF repository. The updated PTF will remove the strict dependencies and allow updates to be installed again.
+
+The replacement of the PTF with the maintenance update which includes the fix happens automatically via
+a standard package update or patch installation.
+
+
+== Removing the patched version of a package
+
+In cases where a PTF needs to be uninstalled and the non-patched version of the packages need to be installed on the system,
+a simple package remove cannot be used. The ptf package is not selectable in the standard package list page.
+
+. In the {productname} {webui}, navigate to menu:Systems[Systems List] and select the client you want to remove a PTF from.
+. Navigate to the menu:Systems[Software > PTFs > List/Remove] sub tab.
+. Select the ptf package you want to remove.
+. Click btn:[Remove PTFs], and on the [guimenu]``Confirm Program Temporary Fixes (PTFs) Removal`` page click btn:[Confirm].
+. To see the results, navigate to menu:Systems[Events > History].
+
+
+[NOTE]
+====
+Removing PTFs requires a special version of [literal]``libzypp`` and [literal]``zypper`` to be installed on the client system.
+Check [command]``zypper --help`` whether [command]``removeptf`` is supported.
+The ``List/Remove`` tab is only visible if this condition is met.
+====
+
+In case the PTF should be removed using the API, the normal [systemitem]``system.schedulePackageRemove`` api can be used with the proxy package name.
+
+
+== Removing the patched version of a package on the client
+
+In case a PTF should be removed directly on the client using the console, it is required to use a special command ``zypper removeptf ...``.
```

Comment:
```suggestion
In case a PTF should be removed directly on the client using the console, it is required to use a special command [literal]``zypper removeptf ``.
```